assets/src/console/pages/services/TeamInvitePage.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import { Helmet } from "react-helmet";
+import {
+  PageTitle,
+  Radio,
+  BasicTextInput,
+  Field,
+  Button,
+  Form,
+  Fieldset,
+  LinkButton,
+  ErrorAlert
+} from "@pay/web";
+
+import { BreadBox } from "@pay/web/components/Breadcrumb";
+import { useInviteUserMutation } from "../../__generated__/graphql";
+import { Redirect } from "react-router-dom";
+
+export interface props {
+  service: { id: string; externalId: string; name: string };
+  roles: { description: string; name: string }[];
+}
+
+interface FormValues {
+  email: string;
+  role: string;
+}
+
+const roleDescriptions = {
+  super_admin: (
+    <>
+      <div>View transactions</div>
+      <div>Refund payments</div>
+      <div>Manage settings</div>
+    </>
+  ),
+  admin: (
+    <>
+      <div>View transactions</div>
+      <div>Refund payments</div>
+      <div>Manage settings</div>
+    </>
+  ),
+  view_and_refund: (
+    <>
+      <div>View transactions</div>
+      <div>Refund payments</div>
+      <div>Cannot manage settings</div>
+    </>
+  ),
+  view_only: (
+    <>
+      <div>View transactions</div>
+      <div>Cannot refund payments</div>
+      <div>Cannot manage settings</div>
+    </>
+  )
+};
+
+const RoleDescription: React.FC<{ role?: string }> = ({ role }) => {
+  return roleDescriptions[role as keyof typeof roleDescriptions] || null;
+};
+
+const TeamInvitePage: React.FC<props> = ({ service, roles }) => {
+  const [
+    inviteUser,
+    { data, loading, error: inviteError }
+  ] = useInviteUserMutation({
+    refetchQueries: ["GetServiceWithGatewayAccounts"]
+  });
+
+  return (
+    <>
+      <Helmet>
+        <title>Team members - {service.name}</title>
+      </Helmet>
+      <PageTitle
+        title="Invite a team member"
+        breadcrumbs={BreadBox.InviteTeamMember({ service })}
+      />
+
+      {inviteError ? (
+        <ErrorAlert
+          title="Error inviting user"
+          message={inviteError && inviteError.message}
+          showError
+        ></ErrorAlert>
+      ) : null}
+
+      <Form<FormValues>
```

Comment:
did you want any form validation on this form yet, or later?

Reply:
will do later